grabbit/src/main/groovy/com/twcable/grabbit/ClientJobStatus.groovy

```diff
@@ -82,4 +100,12 @@ class ClientJobStatus {
             get(explorer, id)
         }
     }
+
+    static List<ClientJobStatus> getAllForTransaction(@Nonnull final JobExplorer explorer, @Nonnull final long transactionID) {
+        if(!explorer) throw new IllegalArgumentException("jobExplorer == null")
+        if(!transactionID) throw new IllegalArgumentException("transactionID == null")
```

Comment:
While you can't do a null check on a `long` (except via boxing), you can check for `0`. So what that comparison is actually doing is not a null-check as the exception says, but a comparison against `0`. I'm pretty sure that's not what you intended. (And yet another reason being explicit is a good thing...)